app/views/notification_mailer/info_request_batches/_info_request_batch.text.erb

```diff
@@ -0,0 +1,14 @@
+<%= info_request_batch.title %>
+<%= render partial: 'alaveteli_pro/info_request_batches/authority_list',
+           locals: { public_bodies: info_request_batch.public_bodies } %>
+<%= render partial: 'alaveteli_pro/info_request_batches/progress_bar',
+          locals: { phases: info_request_batch.request_phases_summary,
+                    batch: info_request_batch } %>
+<%= _("What's happened:") %>
+<% notifications_by_event_type = notifications.group_by { |n| n.info_request_event.event_type } %>
```

Comment:
Done in 8940262 as a separate commit for introducing the helper and 0a088e7 to fix this up to use it. I'll shuffle the commits around when I squash this down.